crml/staking/src/lib.rs

```diff
@@ -621,6 +621,21 @@ impl Default for Forcing {
 	}
 }
 
+// A value placed in storage that represents the current version of the Staking storage. This value
+// is used by the `on_runtime_upgrade` logic to determine whether we run storage migration logic.
+// This should match directly with the semantic versions of the Rust crate.
+#[derive(Encode, Decode, Clone, Copy, PartialEq, Eq, RuntimeDebug)]
+enum Releases {
+	V0 = 0,
+	V1 = 1,
```

Comment:
could you add a comment about the runtime versions for V0 and V1

Reply:
Sure